_examples/recordio.go

```diff
@@ -21,7 +22,7 @@ func main() {
 }
 
 func simpleRead(path string) {
-	reader, err := rProto.NewProtoReaderWithPath(path)
+	reader, err := rProto.NewReader(rProto.ReaderPath(path))
```

Comment:
good catch, I think there is also a reference to it in the examples in the respective READMEs - if you could also sweep those with a search/replace please